client/driver/rkt.go

```diff
@@ -731,6 +951,36 @@ func (d *RktDriver) Open(ctx *ExecContext, handleID string) (DriverHandle, error
 	return h, nil
 }
 
+func (d *RktDriver) fetchImage(img lastditch.RuntimeImage, fetchArgs []string) (string, error) {
+	var version string
+	for _, label := range img.Labels {
+		if label.Name == "version" {
+			version = label.Value
+			break
+		}
+	}
+
+	var imgName string
+	if version == "" {
+		return "", fmt.Errorf("no version supplied for image: %s", img.Name)
+	}
+
+	imgName = fmt.Sprintf("docker://%s:%s", img.Name, version)
```

Comment:
If we do this, we'd only support docker images (converted via `docker2aci`) so regular ACI images won't be supported. For example: `rkt run coreos.com/etcd:v2.0.10` works, but if we use `coreos.com/etcd` in a pod manifest, it will try to fetch `docker://coreos.com/etcd`, which will fail.

The problem is that we can't use the syntax `docker://` in `imgName` with the regular appc schema PodManifest. However, we're now using `lastditch` so that should be possible right?

If that's possible, I suggest deviating again from the appc spec and accept image names like `docker://`, passing that directly to rkt.